examples/proactive/assets/inject.js

```diff
@@ -0,0 +1,47 @@
+function injectDOMElement(tagName, targetSelector, options) {
+  const element = document.createElement(tagName)
+  if (options) {
+    Object.keys(options).forEach(function(key) {
+      element[key] = options[key]
```

Comment:
I don't know what's in options but I feel like you could simply do the following

```
element = {...element, ...options}
```

Reply:
Not possible, since that JS needs to be vanilla so IE users can run it ! (No spread, no arrow... dull world!)